docs/native/src/index.rst

```diff
@@ -1,8 +1,8 @@
 
 ..  _native wopi:
 
-Using the WOPI protocol to integrate with Office for iOS
-========================================================
+Using the WOPI protocol to integrate with Office for iOS and Android
```

Comment:
You can actually use the `|Office iOS Android|` placeholder in headings as well.

Reply:
Will update